behat/symfony2-extension/2.1/behat.yaml.dist

```diff
@@ -0,0 +1,7 @@
+default:
+    extensions:
+        Behat\Symfony2Extension:
+            kernel:
+                bootstrap: vendor/autoload.php
```

Comment:
instead of creating a file with only dotenv loading, why not prefix the command with `source .env`?